samples/wifi/provisioning/ble/Kconfig.sysbuild

```diff
@@ -0,0 +1,13 @@
+#
+# Copyright (c) 2023 Nordic Semiconductor
+#
+# SPDX-License-Identifier: LicenseRef-Nordic-5-Clause
+#
+
+source "${ZEPHYR_BASE}/share/sysbuild/Kconfig"
+
+config NRF_DEFAULT_IPC_RADIO
```

Comment:
Do we need any other settings for IPC_RADIO, like the ones done https://github.com/nrfconnect/sdk-nrf/tree/main/samples/openthread/cli/sysbuild/ipc_radio ?

Reply:
They are in place already.